opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -765,6 +765,18 @@ def save_as_pt(
 
         # save tokenizer.json in save_json_folder_name
         model.save(save_json_folder_path)
+        with open(save_json_folder_path + "/tokenizer.json") as user_file:
+            file_contents = user_file.read()
+        parsed_json = json.loads(file_contents)
+        if not parsed_json["truncation"]:
```

Comment:
Can we do `if "truncation" not in parsed_json or parsed_json["truncation"] is None`? I think we should handle the case where `"truncation"` is not in `parsed_json` similar to when `parsed_json["truncation"]` is `None` just in case.

Reply:
Done